compiler/rustc_feature/src/unstable.rs

```diff
@@ -555,6 +555,8 @@ declare_features! (
     (unstable, offset_of_nested, "CURRENT_RUSTC_VERSION", Some(120140)),
     /// Allows using `#[optimize(X)]`.
     (unstable, optimize_attribute, "1.34.0", Some(54882)),
+    /// Allows using pattern types.
+    (internal, pattern_types, "CURRENT_RUSTC_VERSION", Some(54882)),
```

Comment:
Shouldn't this at the top with the others "internal feature gates (no tracking issue)"? Seems to be copy-pasted without changing the attached issue.

Reply:
Yes please